src/pages/Home.tsx

```diff
@@ -14,7 +14,7 @@ export const HomePage: React.FC = () => {
 
   return (
     <>
-      <Header onSearch={setSearchTerm} />
+      <Header onSearch={setSearchTerm} showSearch ={true}/>
```

Comment:
Nit. can we remove the space here ?
```suggestion
      <Header onSearch={setSearchTerm} showSearch={true}/>
```

Reply:
fixed https://github.com/Sakshi-Simform/Recipe-App/pull/4/commits/edcbae0794aac5d53bc4d1bd22a792293df4e006